src/main/java/org/jenkinsci/plugins/badge/StatusImage.java

```diff
@@ -34,7 +35,7 @@
     /**
      * TO DO
      */
-    private final byte[] payload;
+    public final byte[] payload;
```

Comment:
My bad! I was testing something out, and forgot to undo it. It should be private!